dart/lib/src/attachment.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:typed_data';
+export 'attachment_extensions/attachment_extensions.dart';
+
+// https://develop.sentry.dev/sdk/envelopes/#attachment
+
+/// Arbitrary content which gets attached to an event.
+class Attachment {
+  Attachment({
+    required this.content,
+    required this.fileName,
+    AttachmentType? type,
+    String? mimeType,
+  })  : mimeType = mimeType ?? 'application/octet-stream',
```

Comment:
This probably shouldn't default to `application/octet-stream`, but to `null` in order to be inferred in the backend.

Reply:
`h`: Can we guess the MIME content type in dart based on the filename, as described here: https://develop.sentry.dev/sdk/features/#attachments

> and guess the content type of the attachment via the filename.

Reply:
We set the default in Java and Objective-C, but you are right; we could remove them and let the backend decide.

Reply:
For reference we had a conversation [here](https://discord.com/channels/621778831602221064/654883298736340994/856597058093187152) on Discord about that.